packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -8,111 +8,58 @@ $Rating-disabledColor: $ms-color-neutralTertiaryAlt;
 $Rating-smallIconSize: 16px;
 $Rating-largeIconSize: 20px;
 
-// Hide input field
-.input {
-  position: absolute;
-  opacity: 0;
-  top: 0px;
-}
-
-.container {
-  position: relative;
+.ratingStar {
   display: inline-block;
+  position: relative;
+}
 
-  &:hover {
-    .star:not(.starIsDisabled) {
-      color: $Rating-selectedColor;
-
-      @include high-contrast {
-        color: HighlightText;
-        background-color: Highlight;
-        -ms-high-contrast-adjust: none;
-      }
-
-      &.star:hover {
-        color: $Rating-hoverColor;
-
-        @include high-contrast {
-          color: HighlightText;
-        }
-
-        & ~ .star {
-          // Set default color for any star after hovered one
-          color: $Rating-defaultColor;
-
-          @include high-contrast {
-            color: GrayText;
-            background-color: Window;
-          }
-        }
-      }
-    }
-  }
-
-  .star {
-    display: inline-block;
-
-    text-align: center;
-
-    color: $Rating-defaultColor;
+.ratingStarBack {
+  color: $Rating-defaultColor;
+  width: 100%;
 
+  &.ratingStarDisabled {
+    color: $Rating-disabledColor;
     @include high-contrast {
-      color: WindowText;
-      -ms-high-contrast-adjust: none;
-    }
-
-    &.starIsSelected {
-      color: $Rating-selectedColor;
-
-      @include high-contrast {
-        color: HighlightText;
-        background-color: Highlight;
-        -ms-high-contrast-adjust: none;
-      }
+      color: GrayText;
     }
+  }
+}
 
-    &.starIsDisabled {
-      color: $Rating-disabledColor;
-
-      @include high-contrast {
-        color: GrayText;
-        background-color: Window;
-      }
-
-      .label {
-        cursor: default;
-      }
-    }
-
-    .label {
-      display: inline-block;
-      cursor: pointer;
```

Comment:
Add `cursor: pointer;` back in on the main star element so that the user doesn't get the text entry cursor. Also, set the cursor back to default when the component is disabled.